api/v1beta2/foundationdbcluster_types.go

```diff
@@ -918,6 +918,9 @@ type FoundationDBClusterAutomationOptions struct {
 
 	// MaintenanceModeOptions contains options for maintenance mode related settings.
 	MaintenanceModeOptions MaintenanceModeOptions `json:"maintenanceModeOptions,omitempty"`
+
+	//IgnoreLogGroupsForUpgrade defines the list of LogGroups that should be ignored during fdb version upgrade.
```

Comment:
```suggestion
	// IgnoreLogGroupsForUpgrade defines the list of LogGroups that should be ignored during fdb version upgrade.
```

Can we add a limitation to the max items of this slice?